source/episodes/002.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 date: 2015-07-01
 excerpt: In this inaugural episode (pay no attention to the episode number behind the curtain), Pat and Sam discuss the first episode of Mr Robot. Critics are raving, but what think our intrepid hosts? Listen to find out!
-title: Mr Robot S01E01
+title: Unnatural Pause
```

Comment:
Maybe I'm just too boring (or literal), but `Mr Robot S01E01` seemed _just perfect_ to me.

I'm torn now between taking this as-is or merging the two: `Unnatural Pause - Mr Robot S01E01`, in the interest of making the title both funny/memorable _and_ useful when looking back on older content. Though, perhaps, the better way of doing that is a show index (a la, all the Mr Robot episodes, all the Game of Thrones episodes, and such), which I think makes sense when we have many more episodes than what we have now.